certified-connectors/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#ededed",
+    "capabilities": [],
+    "publisher": "Evith Techology"

```

Comment:
Please add `stackOwner` property as well

Documentation - [Adding metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

Reply:
Hi, I resolved this one.
Thank you :)